rollup.config.js

```diff
@@ -21,6 +23,14 @@ export default {
     babel({
       exclude: 'node_modules/**'
     }),
+    resolve({
+      customResolveOptions: {
+        moduleDirectory: 'node_modules'
+      }
+    }),
+    commonjs(),
```

Comment:
Is this necessary?  In the output above we're writing in UMD format.  Is this a conflict?

Reply:
Typically, you would use this plugin alongside rollup-plugin-node-resolve, so that you could bundle your CommonJS dependencies in node_modules - from RollUp

Reply:
It works with `iife` so I doubt it wont be compatible with `umd` as well https://github.com/rollup/rollup-plugin-commonjs